Scripts/Modules/IntersectionModule/IntersectionModule.cs

```diff
@@ -74,9 +74,9 @@ void Update()
 				{
 					var intersectionFound = false;
 					m_Intersections.Clear();
-					if (m_SpatialHash.GetIntersections(m_Intersections, tester.renderer.bounds))
+					if (m_SpatialHash.GetIntersections(m_Intersections, tester.collider.bounds))
```

Comment:
They're no longer on the same object. The collider is actually new, so it will technically cost us its bounds computation and whatever physics overhead we get from moving static colliders around in edit mode (really not sure what's involved on that one).

It's necessary because we need direct hover info even while the cones are invisible, and wouldn't be giving us proper bounds information (scaled down and probably not updated when inactive).

There is now a MeshCollider with the cone as its mesh that is used for giving us the AABB for coarse testing as well as the mesh reference for testing each ray.